electron-builder.json

```diff
@@ -1,10 +1,10 @@
 {
   "publish": [{
     "provider": "generic",
-    "url": "https://releases.mattermost.com/desktop/"
+    "url": "http://desktop-test-autoupdater.s3-website-us-east-1.amazonaws.com/"
```

Comment:
0/5, but would it actually make sense to expose a CLI flag or something to override the the update server URL? I'm sure we'll have to test things again at some point even after this has been merged.

Reply:
I like the idea, but I can't find a good place to put this value that wouldn't be easier than just replacing it on my local machine or creating a PR with the changed value. It would have to sit in every CircleCI context or just live in code anyways. Might make more sense to do when we migrate everything to GitLab :)